src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -241,8 +241,8 @@
 						<Grid
 							x:Name="HeaderGrid"
 							Height="40"
-							Margin="0"
-							Padding="16,0,0,0"
+							Margin="-8,0,-8,0"
```

Comment:
Is the negative margin on the right actually doing anything?

Reply:
I think it effects when the horizontal scrollbar is visible, other than that i dont think so.